src/renderer/coremods/badges/index.tsx

```diff
@@ -105,35 +105,33 @@ export async function start(): Promise<void> {
         return res;
       }
 
+      const badgeTypes = [
+        { type: "booster", component: Badge.Booster },
+        { type: "hunter", component: Badge.BugHunter },
+        { type: "contributor", component: Badge.Contributor },
+        { type: "developer", component: Badge.Developer },
+        { type: "early", component: Badge.EarlyUser },
+        { type: "staff", component: Badge.Staff },
+        { type: "support", component: Badge.Support },
+        { type: "translator", component: Badge.Translator },
```

Comment:
These should be reordered. I think it should be:
- staff
- support
- developer
- contributor
- translator
- bug hunter
- booster
- early user